app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Welcome to MiniShop!</h1>
+
+<ul>
+  <li><a href="/merchants">Merchant Index</a></li>
```

Comment:
anytime you need to create a link to something, use actionview helpers since rails provides them for us.. this way you follow rails convention. The action view helper for an anchor tag (that <a> you have there) is `link_to` which you already have some experience using in this project.